src/core/mount.c

```diff
@@ -222,6 +330,8 @@ static void mount_done(Unit *u) {
 
         m->where = mfree(m->where);
 
+        if (mount_hash_remove(m) < 0)
+                log_error("Failed to remove mount hash entry");
```

Comment:
display error codes that are received, i.e. 

```
r = mount_hash_remove();
if (r < 0)
        log_error_errno(r, "Failed to …: %m");
````

Reply:
hmmm, actually, how can mount_hash_remove() ever fail? I figure that function should return void